src/models/Scholarships.test.ts

```diff
@@ -204,3 +204,11 @@ test('scholarships.list - filters by maxAmount', async () => {
     want.map(extractName).sort()
   );
 });
+
+test('dateAdded & lastModified gets set on saving a new scholarship', async () => {
```

Comment:
```suggestion
test('save new scholarship - dateAdded and lastModified get set', async () => {
```

Another test to add:
```suggestion
test('save updated scholarship - dateAdded unchanged but lastModified set', async () => {
```

Reply:
Might have to change where these tests are based on other comments.